src/components/generics/GenericFilter.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+import { toRef, watch } from 'vue'
+import { useDebounceFn } from '@vueuse/core'
+
+export interface Props {
+  checkbox: boolean
+  name: string
+  tooltip?: string
+  checkboxLabel: string
+  rangeSlider?: number[]
+  rangeSliderMin?: number
+  rangeSliderMax?: number
+  rangeSliderLabelPrepend?: number | string
+  rangeSliderLabelAppend?: number | string
+}
+
+const { rangeSlider } = defineProps<Props>()
+const emit = defineEmits(['update:checkbox', 'clickedCheckbox', 'update:rangeSlider'])
+
+const internalRangeSlider = toRef(rangeSlider)
```

Comment:
I encountered A Problem with the handling of the RangeSlider Values.
The InternalRangeSlider Values seem not to be updated when changing the corresponding Values in the FilterStore eg. when you "clear" the Filters.

you can reproduce it by activating a filter, slide to a value, clear the active filters and reactivate the Filter.
The Slider will show the previous Values but the Filter will apply the initial default values:
![Screenshot 2023-08-04 144930](https://github.com/FloKaemmerer/dynamic-atlas/assets/20900866/f7b3c8d3-10ce-4e4c-be7f-4e12b17c24ea)

Reply:
Nice catch, I'll look into it

Reply:
Should deactivating a filter and reenabling it reset the filter also ?

Reply:
Not sure about that one.

But I'd tend to reset it.